Cloud Release Notes since October 2024 (#7033)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/overview-2024/2024-10-releasenotes.md

# October 2024

## Key Takeaways

* **Follow Automatic Upgrades in Real Time** - Follow upgrade progress directly in the portal.

## Follow Automatic Upgrades in Real Time

> Automatic upgrades are now surfaced as part of Long Running Activities
> in the Umbraco Cloud portal.
> This means you can follow along as each of your projects is being upgraded
> – directly from the portal.

Reviewer: ```suggestion
Automatic upgrades are now displayed as part of Long Running Activities in the Umbraco Cloud portal.
This allows you to monitor the upgrades of each of your projects directly from the portal.
```
